sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/implementation/StorageAllowedHeadersAndQueries.java

```diff
@@ -0,0 +1,104 @@
+package com.azure.storage.common.implementation;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class StorageAllowedHeadersAndQueries {
+    public static class BlobHeadersAndQueries {
+        private static Set<String> blobHeaders = Stream.of(
```

Comment:
static fields should be all upper-case, e.g. BLOB_HEADERS

Reply:
Done